components/user/home-screen/Post.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {Image, StyleSheet, View, Text, TouchableOpacity} from 'react-native';
+import {Card} from '@rneui/themed';
+import {MonoText, TextSemiBold} from '@/components/StyledText';
+import Tag from './Tag';
+
+const CONTENT_LENGTH = 200;
+
+type PostProps = {
+  date: string; //TODO: or date object?
```

Comment:
It's best practice to convert to date object, especially if we are going to use this more in the future.